tests/test_intervaldata.py

```diff
@@ -0,0 +1,196 @@
+import random
+
+import numpy as np
+import pytest
+
+from eitprocessing.datahandling.intervaldata import Interval, IntervalData
+
+
+@pytest.fixture()
+def intervaldata1():
+    """Creates an IntervalData object with n sequential intervals lasting 1 second each."""
+    n = random.randint(50, 150)
```

Comment:
why not just fix the `n` in these tests?
Same question for the values in the fixtures below.

Reply:
You are probably correct that it does not matter in this case. I have had some situations in the past where the tests worked fine because of the specific length of the object I was testing. However, there are probably very little side effects with a fixed length.

I made them all different lengths, though, to prevent accidents.